### ISSUE TITLE
Add on/off switch and remote start button

### ISSUE THREAD

**Issue by gabi1324** [rowysock/home-assistant-HomeWhiz#120]
Hi! Today I noticed that this integration doesn't provide an on/off switch or a remote start button. It would be nice to have these controls. 
If it helps, I have a Grundig wifi dishwasher.

**Comment by elektrolitr** [comment]
This functionality is not available for dishwasher in the Homewhiz app, too

To be able to remotely control the appliance, you have to press button on it and then activate the remote control. I suspect it's due to security reasons (like avoid washing your pet) because my Samsung devices (washing machine and dryer) behave exactly the same.

**Comment by gabi1324** [comment]
> To be able to remotely control the appliance, you have to press button on it and then activate the remote control.

 Yes, I know that. Still we can have a remote start button that becomes available in HA when you press the button on the appliance.
And about the on/off switch. If I recall correctly, when you turn it on manualy you have the option to turn it off from the app. When you turn it off with remote control on, with the app, you can turn it back on from the app. 

> I suspect it's due to security reasons (like avoid washing your pet) because my Samsung devices (washing machine and dryer) behave exactly the same.

My LG washing machine and dryer work the same (I have to turn them on and press the remote start button manualy before I can use the app). The LG integration with HA provides an on/off switch and a remote start button that become available after I turn on and press the button on the device. 

In theory it's posible to have the on/off switch and remote start button, but I don't know if the API supports these actions.

**Comment by thecrisb** [comment]
As long as remote control has been enabled on mt Beko washing machine I am able to fully configure the cycle and then start the the machine status to "DEVICE_STATE_RUNNING".

![DrumClean](https://github.com/rowysock/home-assistant-HomeWhiz/assets/19705579/c42ab71c-a17b-40ff-ae62-d1adfb6fceaf)
![Homewizz Start](https://github.com/rowysock/home-assistant-HomeWhiz/assets/19705579/0391f457-9f4c-4bbd-8820-57a3330984db)
![Running](https://github.com/rowysock/home-assistant-HomeWhiz/assets/19705579/42f06c5e-01db-45c4-8427-99095640fb65)

Although the washer doesn't seem to respond fast enough to activate all setting using a scene a script to ease adjusting the setting through an automation seams to perform adequately.

**Comment by TechHummel** [comment]
@gabi1324 I'm afraid I'm currently quite busy 😞 and I would like to have a look at all new issues first 👀 . Also, I personally feel that the translations should be fixed before a new version is released 👍
